app/src/main/java/org/ieeeguc/ieeeguc/models/type.java

```diff
@@ -0,0 +1,11 @@
+package org.ieeeguc.ieeeguc.models;
+
+/**
+ * Created by sally on 31/01/2017.
+ */
+
+public enum type {
+    Member , admin , systemAdmin;
```

Comment:
Capitalize all letters and for the please read the documentation carefully for the account types